pkg/codegen/go/gen.go

```diff
@@ -2493,6 +2508,21 @@ func (pkg *pkgContext) getTypeImports(t schema.Type, recurse bool, importsAndAli
 				// tests don't include ImportBasePath
 				goInfo.ImportBasePath = extractImportBasePath(extPkg)
 			}
+
+			// Ensure that any package import aliases we have specified locally take precedence over those
+			// specified in the remote package.
+			if ourPkgGoInfoI, has := pkg.pkg.Language["go"]; has {
+				ourPkgGoInfo := ourPkgGoInfoI.(GoPackageInfo)
+
+				for k, v := range ourPkgGoInfo.PackageImportAliases {
+					if goInfo.PackageImportAliases == nil {
```

Comment:
Does it matter if this is set to non-nil map if `ourPkgGoInfo.PackageImportAliases` is empty? It's odd its in the loop rather than just before and I'm not sure if that's because its important to be nil or not?

Reply:
It's not import to be nil or not -- it's just to avoid unnecessarily creating the map. I've moved this outside the loop.